spras/analysis/cytoscape.py

```diff
@@ -33,25 +35,28 @@ def run_cytoscape(pathways: List[Union[str, PurePath]], output_file: str, contai
     volumes.append((cytoscape_output_dir, PurePath(work_dir, 'CytoscapeConfiguration')))
 
     # Map the output file
-    bind_path, mapped_output = prepare_volume(output_file, work_dir)
+    bind_path, mapped_output = prepare_volume(output_file, work_dir, container_settings)
     volumes.append(bind_path)
 
     # Create the initial Python command to run inside the container
     command = ['python', '/py4cytoscape/cytoscape_util.py', '--output', mapped_output]
 
     # Map the pathway filenames and add them to the Python command
     for pathway in pathways:
-        bind_path, mapped_pathway = prepare_volume(pathway, work_dir)
+        bind_path, mapped_pathway = prepare_volume(pathway, work_dir, container_settings)
         volumes.append(bind_path)
         # Provided the mapped pathway file path and the original file path as the label Cytoscape
         command.extend(['--pathway', f'{mapped_pathway}|{pathway}'])
 
+    out_dir = Path(output_file).parent
```

Comment:
Where should these log files go for Cytoscape? It isn't a pathway algorithm, so it's out_dir isn't self-contained like the others are.